es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst

```diff
@@ -27,15 +27,15 @@ Todas las *releases* actuales están alojadas en CakeForge. Este *site*
 también contiene enlaces a muchos otros proyectos en CakePHP, incluyendo
 *plugins* y aplicaciones para CakePHP. Las *releases* de CakePHP estásn
 disponibles en
-`http://cakeforge.org/projects/cakephp <http://cakeforge.org/projects/cakephp>`_.
+`https://cakeforge.org/projects/cakephp <https://cakeforge.org/projects/cakephp>`_.
 
 Se crean *nightly builds* alternativas que incluyen parches y mejoras al
 minuto (bueno, al día). Estas pueden ser accedidas desde la página
 principal de descargas aquí:
 `https://cakephp.org/downloads/index/nightly <https://cakephp.org/downloads/index/nightly>`_.
 Para actualizaciones realmente al minuto, puedes obtener el código
 directamente de la rama de desarrollo del repositorio git aquí:
-`http://code.cakephp.org/source <http://code.cakephp.org/source>`_.
+`https://code.cakephp.org/source <https://code.cakephp.org/source>`_.
```

Comment:
This link is invalid. I'm not sure where it came from @markstory

Reply:
code.cakephp.org predates our usage of github. There was a brief period in the projects history where we were hosted in a custom git server that one of the former contributors wrote.

Reply:
remove